### ISSUE TITLE
Laravel 5.3 support

### ISSUE THREAD

**Issue by patrickguevara** [mmieluch/laravel-vfs-provider#2]
Running into these problems when upgrading to Laravel 5.3

```
mmieluch/laravel-vfs-provider 0.0.2 requires illuminate/support 5.0.*|5.1.*|5.2.*
mmieluch/laravel-vfs-provider 0.0.2 requires illuminate/filesystem 5.0.*|5.1.*|5.2.*
```

**Comment by patrickguevara** [comment]
I'm using a forked copy of this and it seems to be working just fine (all my previous tests are still passing)

```
"repositories": [
    {
        "type": "vcs",
        "url": "https://github.com/metricloop/laravel-vfs-provider"
    }
]
```

The only difference is adding `5.3.*` to `composer.json` that requires Illuminate\Support and Illuminate\Filesystem.

**Comment by mmieluch** [comment]
Thanks for the #3 PR Patrick! Your commit is merged and tagged as 0.0.3. Packagist already picked up the changes. Happy coding!
